### ISSUE TITLE
Edit.com (and possibly other graphical DOS programs) do not work at all on latest Windows 10 insider build

### ISSUE THREAD

**Issue by kristibektashi** [leecher1337/ntvdmx64#116]
Note: I also have Avast Free Antivirus installed although I disabled before launching the autobuild script (it got re-enabled after reboot)

**Comment by leecher1337** [comment]
Cannot reproduce, updated Win10 20H2 to lates Dev Insider preview and then installed NTVDMx64, works fine here (WOW32 and DOS):

![image](https://user-images.githubusercontent.com/6495842/107854735-41fb6200-6e1e-11eb-80d9-7c1cdaf25c83.png)

**Comment by kristibektashi** [comment]
@leecher1337 I have noticed another issue that is possibly exclusive to my system. When I update to a newer Insider build, the NTVDMx64 files are missing on system32 folder but not syswow64 and I have to reinstall NTVDMx64 (I didn't uninstall/reinstall. I simply reinstalled it on top of the previous version)

**Comment by kristibektashi** [comment]
Maybe try installing NTVDMx64 BEFORE upgrading to latest insider preview.

Note: I am using Greek Windows 10 but Greek language isn't supported on NTVDMx64 so I installed usa version instead

**Comment by leecher1337** [comment]
I always unsinstall NTVDMx64 before updating Windows. One reason is that the loader locks i.e. conhost.exe and may prevent updates from being installed correctly, making the update fail. Therefore, uninstall NTVDMx64 via control panel, update and then reinstall may be the best strategy for updates. Windows sometimes cleans out the Windows folder on major updates, which also ruins other applications that place files in the Windows directory.
The installer has flags that tell the system to replace files that are still in use (the loaders) on next reboot, still, I prefer uninstalling and reinstalling properly with reboots in between to ensure that the loaders get replaced correctly.

**Comment by kristibektashi** [comment]
Nope, it still doesn't work, every time I install NTVDM, WOW32 works but DOS apps don't. I tried uninstalling NTVDM, installing the new Insider build and reinstalling NTVDM but it still doesn't work

**Comment by leecher1337** [comment]
WOW32 works... hmm, then you need the DebugView output from the loader, could i.e. be a symbol lookup problem from Microsoft Symbol Servers that prevents proper patching of all functions. Maybe symbols for your Operating System release are missing on Microsoft Symbol Server and they forgot to add it. This already happened once iirc.
So please paste DebugView output when launching DOS application here.

**Comment by kristibektashi** [comment]
> WOW32 works... hmm, then you need the DebugView output from the loader, could i.e. be a symbol lookup problem from Microsoft Symbol Servers that prevents proper patching of all functions. Maybe symbols for your Operating System release are missing on Microsoft Symbol Server and they forgot to add it. This already happened once iirc.
> So please paste DebugView output when launching DOS application here.

How do I do that?

**Comment by leecher1337** [comment]
Start https://docs.microsoft.com/en-us/sysinternals/downloads/debugview and watch output when starting a DOS program, the loader normally tells you what it does

**Comment by kristibektashi** [comment]
I have lost access to the system that this issue occurred at originally so I can't provide any more information for it, but I am reopening this since a solution was never found just in case anyone else stumbles into the same issue
